apps/src/aichat/views/modelCustomization/PromptCustomization.tsx

```diff
@@ -35,6 +43,33 @@ const PromptCustomization: React.FunctionComponent = () => {
   return (
     <div className={styles.verticalFlexContainer}>
       <div>
+        <div className={classNames(styles.inputContainer)}>
+          <SimpleDropdown
+            labelText="Pick a model:"
```

Comment:
If this field is read-only, would we want the user still see 'Pick a model' label above?
Generally speaking, what is the plan for read-only vs editable fields?

Reply:
mmm good q...I basically deferred on all of the levelbuilder-related stuff in this PR, but I see [this comment from Dan](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1711478711932649?thread_ts=1711473774.746449&cid=C06FELVTV0Q) -- maybe we show the dropdown (disabled) but no label (we'd want to be careful with for a11y though)

Reply:
Yeah - that is a useful thread to reference - thanks!
@dancodedotorg said:
> If read-only, I'd expect the dropdown to be visible but "Compare Models" button to be hidden or disabled.

So it seems like if it's read-only, we'd want the label above the dropdown to perhaps be: 'Selected model' or 'Model'? @dancodedotorg let us know what you think.

This question is not for this PR but follow-up work.

Reply:
Asking about this in Slack and tagged Alice!

Reply:
Just documenting that Sam and Dan prefer `Selected model' - see thread above.

Reply:
Updated!